### ISSUE TITLE
the value of 'm‘ in “Batched Adversarial Attack”  of your paper

### ISSUE THREAD

**Issue by luni-chen** [cdluminate/robrank#16]
Excuse me, what's the value of 'm‘ in “Batched Adversarial Attack”  of your paper? Could you give a detailed guidance to reproduce the result of Table in your paper?

**Comment by cdluminate** [comment]
By default `-m` is not specified. In that way, the algorithm will go through the whole dataset. The step-wise, detailed guidance is written in README.md -- please be specific about the part which you don't understand. Meanwhile, this code base is connected with two papers, please be specific about the name of the paper.

**Comment by cdluminate** [comment]
In the paper you can find that the algorithm will go through the whole dataset for reporting the number. `-m` is mostly used for debugging, and can be directly omitted.

**Comment by luni-chen** [comment]
ok,thanks a lot. I want to reproduce the result of 《Enhancing Adversarial Robustness for Deep Metric Learning》. However, it is relatively different from the results of the paper? Is it normal?
![image](https://user-images.githubusercontent.com/55188223/198861606-f826686e-0a03-438c-a463-e376c5320923.png)

**Comment by luni-chen** [comment]
I follow the step of README.md, I think there's no wrong step.  And the result is trained on CUB dataset

**Comment by cdluminate** [comment]
This difference is normal. What you see here is within the error bar. Due to different initialization and other factors (such as the number of GPUs in DDP mode), the performance differs slightly. If you want to see a higher ERS, just try to limit the GPU number to 1 or 2, and try some more initialization.

**Comment by luni-chen** [comment]
ok ,I will try it. As to the R1/R2/mAP/NMI,  are they the result of training end? 
![image](https://user-images.githubusercontent.com/55188223/198863139-87d21775-0ba0-487a-9f1b-f67f9109e62c.png)

**Comment by cdluminate** [comment]
Yes, they are reported at the training end status. Because in adversarial training, these standard benign metrics may look like a U-shape curve or directly a descending curve ... That's a part of adversarial training sacrificing the benign performance.

**Comment by luni-chen** [comment]
Get it, thank you very much!

**Comment by luni-chen** [comment]
Excuse me, is the mAP in 《Enhancing Adversarial Robustness for Deep Metric Learning》 mAP or mAP@R?

**Comment by cdluminate** [comment]
It's simply the original mAP. If mAP@R is used, it should have been explicitly justified.
